### ISSUE TITLE
finals2000A.all seems inaccessible using api.load.download("finals2000A.all") 

### ISSUE THREAD

**Issue by randyg3000** [skyfielders/python-skyfield#732]
Very recently, this function started failing. It appears skyfield is looking for

https://datacenter.iers.org/data/latestVersion/finals.all.iau2000.txt

which is currently unavailable. Maybe it is only a temporary glitch, but it seems to be a single point of failure. The only place I've found finals2000A.all is behind an account registration (which is a work-around for now):

https://cddis.nasa.gov/archive/products/iers/finals2000A.all

**Comment by mworion** [comment]
There is a message from them:
[https://www.iers.org/IERS/EN/NewsMeetings/News/news_001.html](url)
Maintenance work. And the earth rotations data should not degrade that fast for general usage.
Michel

**Comment by randyg3000** [comment]
Yes, https://github.com/skyfielders/python-skyfield/issues/730 does help.

Also- I've found that https://maia.usno.navy.mil/ser7/finals2000A.daily.extended

is a good backup source. I think this could be closed with a potential feature request to have backup URLs for the data.

**Comment by brandon-rhodes** [comment]
Thanks, everyone, for opening an issue in my absence (I was traveling to attend PyCon, the big annual Python conference) to collect possible solutions to this problem. Since @aendie says over in #730 that ‘I spoke with someone at IERS in Frankfurt who assured me that the ftp.iers.org server should be the primary choice for downloading EOP data’, I'm not tempted at this point to change the URL that Skyfield uses to fetch the file, but it does seem important to alert users to the fact that there might be a problem fetching it.

I'll take a look at whether the docs should show folks how to write fallback code that tries a second URL if the first fails, but in the meantime, as you can see at the issue linked just above this comment, I've at least edited the docs to point to this issue. Hopefully I'll get a release out late this week so that the docs update and folks can see my edits!

**Comment by rmathar** [comment]
If the ftp server is down, the data in https://datacenter.iers.org/products/eop/long-term/c04_14/iau2000/ seem to contain the parameters in yearly chunks, and all of them since 1962 in the file eopc04_14_IAU2000.62-now . They do not contain predictions into the near future as in the maia.usno files.

**Comment by aendie** [comment]
In my software I use https://maia.usno.navy.mil/ser7/finals2000A.all as an alternate location for IERS EOP data. It contains exactly the same data as in https://datacenter.iers.org/data/9/finals2000A.all ... I just checked.

For me, the predictions (at least for the `Bull. A UT1-UTC (sec. of time)` data item), which currently end 26th August 2023,  in the IERS data are the most significant factor when compiling  Nautical Almanac tables for the near future. Quote from IERS:

> Quick-look weekly estimates of the EOP determined by combining the most recently available observed and modeled data (including VLBI 24-hour and intensive, GPS, and AAM). The combination process involves applying systematic corrections and slightly smoothing, in order to remove the high frequency noise.

Thanks for the alternate link you posted above... in the `/data/9` folder.

**Comment by rmathar** [comment]
There are countries where access to the .mil domains is denied: https://lightrun.com/answers/astropy-astropy-list-of-mirror-urls-for-iers-download

**Comment by brandon-rhodes** [comment]
I should add example code to the documentation, showing how to build a custom timescale with a file pulled from an alternative URL.
